src/ddqa/cli/create/__init__.py

```diff
@@ -22,10 +22,27 @@
     multiple=True,
     help='Labels that will be attached to created issues',
 )
+@click.option(
+    '-i',
+    '--include-labels',
+    'include_labels',
+    required=False,
+    multiple=True,
+    help='Labels that will be attached to created issues',
```

Comment:
This description needs to be modified since it is a copy of the one above

Reply:
🤦